.dependabot/config.yml

```
@@ -0,0 +1,12 @@
+version: 1
+update_configs:
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "live"
```

```suggestion
    update_schedule: "daily"
```
Unfortunately it looks like the only options available for Go are  daily, weekly, monthly.

When I validate the file with https://dependabot.com/docs/config-file/validator/ I get:
> **Validation errors** The property '#/update_configs/0/update_schedule' value "live" did not match one of the following values: daily, weekly, monthly